cob_mimic/src/mimic_node.py

```diff
@@ -60,6 +60,7 @@
 import sys
 import os
 import subprocess
+import vlc
```

Comment:
Actually, we tried doing that but I think some dependencies were not met.
@ipa-nhg please correct me, if I am wrong.